src/main/java/org/example/RockPaperScissors/Processor.java

```diff
@@ -0,0 +1,57 @@
+package org.example.RockPaperScissors;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.io.Serializable;
+
+
+
+
+@Slf4j
+public class Processor  {
+
+    private static final HibernateFacRPS SDA_SESSION_FACTORY
+            = new HibernateFacRPS();
+
+    public  void save() {
+        SessionFactory sessionFactory = SDA_SESSION_FACTORY.getSessionFactory();
+        try (Session session = sessionFactory.openSession()) {
+
+            Player gracz = new Player();
```

Comment:
za polskie nazwy w kodzie bede bic xD 
a tak serio to nie uzywajcie nigdy polskich nazw w kodzie, tak sie po prostu nie pisze. Jedyna dopuszczalna opcja to gdybyscie nazwali tego playera tak jak jego name czyli piotr. Bo to by byla bardziej "nazwa wlasna" niz slowo przetlumaczone z angielskiego na polski